### PR TITLE
Add CCMS caseworker users

### LINKED ISSUE #28: Add caseworker actors/people to CCMS
Current documentation describes the following actors:

- Means caseworker
- Merits caseworker
- Billing caseworker
- Case management

### PR DESCRIPTION
**Pull request by oliverweighell**
## What does this pull request do?

Closes #28 
Addresses part of #56. 

- I've only added Oracle Forms as that seemed the necessary component for representing the CCMS caseworker interaction. I was thinking that the other components of issue 56 could be added as a separate PR.

- I've described the caseworker role in the interaction with Oracle Forms rather than on the person definition in LegalAidAgencyUsers. This is purely because it looks nicer that way! Happy to change. 

- I'm also not entirely clear on the role of the billing case worker here.

## What is the intent behind these changes?

Represent caseworkers on the CCMS container diagram:
structurizr-55246-ccms-context-container

### CODE REVIEW

File: src/main/kotlin/model/LegalAidAgencyUsers.kt [hunk 1]
```diff
@@ -30,6 +32,18 @@ class LegalAidAgencyUsers private constructor() {
         "Verifies legal aid provider's bills"
       )
 
+      meansCaseWorker = model.addPerson(
+        Location.Internal,
+        "Legal aid means case workers",
```
Comment (heidar) [inline src/main/kotlin/model/LegalAidAgencyUsers.kt:37]: Can we add a bit more detail to these perhaps? Or can we refer to them as a team maybe like the others?

Comment (oliverweighell) [reply src/main/kotlin/model/LegalAidAgencyUsers.kt:37]: Sure - do you mean label them as the 'Means caseworker team' or something on those lines? I did initially add a description to the means and merits caseworkers but that adds an extra speech bubble to the CCMS container diagram for each user which shifted the elements into a spaghetti-like form.

Comment (oliverweighell) [reply src/main/kotlin/model/LegalAidAgencyUsers.kt:37]: So instead I added a bit more description to their relationship with Oracle Forms.

**Comment by adrianclay** [comment]
👍 I think modelling EBS as Oracle Forms, for now, is a good first step.

File: src/main/kotlin/model/LegalAidAgencyUsers.kt (hunk 1 above)
Comment (heidar) [reply src/main/kotlin/model/LegalAidAgencyUsers.kt:37]:
```
"Means Case Workers",
"A legal aid case worker who assesses legal aid applications for means"
```

Something like that? :P

Comment (oliverweighell) [reply src/main/kotlin/model/LegalAidAgencyUsers.kt:37]: I found that putting the description here adds user speech bubbles to the container diagrams, which makes the CCMS diagram quite hard to follow, here's an example -

structurizr-55246-ccms-context-container

I'll play around with layout settings and see if that helps, but there is so much going on in the CCMS diagram that I'm not sure having the extra bubble text is going to help.

Comment (oliverweighell) [reply src/main/kotlin/model/LegalAidAgencyUsers.kt]: Yeah I'm not sure it works so well with the bubbles. Is there a way to turn these off for specific diagrams? If not, might be better to do without - with means and merits caseworkers, I'm not sure the added description is needed anyway, it is what it says on the tin?

Comment (heidar) [reply src/main/kotlin/model/LegalAidAgencyUsers.kt]: Hmm the bubbles are only on the pUML diagrams though and not on C4? So won't really matter?

Comment (adrianclay) [reply src/main/kotlin/model/LegalAidAgencyUsers.kt]: Shall we add the extra description, and then when we feel the diagrams are more complete we could remove stuff if it makes it messy.

I think we'll get a better idea of how they should be simplified once we have more components/actors/etc visualised.

Comment (oliverweighell) [reply src/main/kotlin/model/LegalAidAgencyUsers.kt]: Ah yes, true about the bubbles. Have added.

**Review by heidar — APPROVED** [review]
